actions/docker-deploy/action.yml

```diff
@@ -0,0 +1,33 @@
+name: Docker Deploy
+description: Docker build, tag and push
+
+inputs:
+  registry:
+    required: true
+    description: Url of the image registry
+
+  repository:
+    required: true
+    description: Name of the repository
+
+  tag:
+    required: false
+    default: dev
+    description: Tag of the built image
+
+  dockerfile:
+    required: false
+    default: dockerfile.yml
```

Comment:
I know this default value is coming from aws-workflows, but it should be `Dockerfile` to match how 99% of all dockerfiles are named on earth :/ If you have other changes to make around here, would be nice to update that 😉